code/modules/events/immovable_rod.dm

```diff
@@ -89,18 +89,16 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 			clong.ex_act(EXPLODE_HEAVY)
 
 /obj/effect/immovablerod/event
-	var/tiles_moved = 0
 	// The base chance to "damage" the floor when passing. This is not guaranteed to cause a full on hull breach.
 	// Chance to expose the tile to space is like 60% of this value.
 	var/floor_rip_chance = 40
 
 /obj/effect/immovablerod/event/Move()
 	var/atom/oldloc = loc
 	. = ..()
-	tiles_moved++
 	if(prob(floor_rip_chance))
 		var/turf/simulated/floor/T = get_turf(oldloc)
 		if(istype(T))
 			T.ex_act(EXPLODE_HEAVY)
-	if(get_dist(oldloc, loc) > 2 && tiles_moved > 10) // We went on a journey, commit sudoku
```

Comment:
I'm trying to wrap my head around this. But does this code make it so that rods "can" stop their travels midway through the station? Since their `x` and `y` coordinate gets set randomly in `obj/effect/immovablerod/Bump(atom/clong)`

Reply:
I am frankly not sure what the original intention of this line was.